secboot/secboot_hooks.go

```diff
@@ -192,18 +196,6 @@ func unlockVolumeUsingSealedKeyFDERevealKeyV2(sealedEncryptionKeyFile, sourceDev
 			}
 		}
 	}()
-	// ensure that the model is authorized to open the volume
```

Comment:
can you put a comment on the PR description why we can drop this? but we  do SkipModel... in the tpm one?

Reply:
We do set the model to `options` instead. Though, we will call `secboot.SetModel` in follow up PRs.